docs/ops/activation/Swish_4.md

```diff
@@ -0,0 +1,49 @@
+## Swish <a name="Swish"></a>
+
+**Versioned name**: *Swish-4*
+
+**Category**: *Activation*
+
+**Short description**: Swish takes one input tensor and produces output tensor where the Swish function is applied to the tensor elementwise.
+
+**Detailed description**: For each element from the input tensor calculates corresponding
+element in the output tensor with the following formula: `features[i] = i * sigmoid(beta * i)`.  
```

Comment:
The formula is wrong. Lets write it like this:

`Swish(x) = x * sigmoid(beta * x)` where `sigmoid(x) = 1.0 / (1.0 + pow(e, -x))`.

Reply:
Also, please, use mathematical formulas like in https://github.com/openvinotoolkit/openvino/pull/1149.

Reply:
Done.